runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -134,6 +134,12 @@ J9::X86::CodeGenerator::initialize()
       cg->setSupportsBDLLHardwareOverflowCheck();
       }
 
+   static char *disableInlineVectorizedMismatch = feGetEnv("TR_disableInlineVectorizedMismatch");
```

Comment:
@0xdaryl I've only considered `arraycmp` support in enabling this acceleration on x. Is there anything else to take into consideration here?